storage-providers/filecoin-deals/filecoin-programs.md

```diff
@@ -52,6 +77,129 @@ Another program that allows storage providers easy access to Fil+ deals is [Big
+
+See the below video for more information on Slingshot:
+
+{% embed url="https://www.youtube.com/watch?v=14sasiEThig" %}
+A overview of the Slingshot project from the Enterprise Storage Provider Accelerator (ESPA).
+{% endembed %}
+
+See also:
+
+- [Slingshot v2](https://v2.slingshot.filecoin.io/)
+- [Slingshot v3](https://v3.slingshot.filecoin.io/)
+- [Slingshot Recovery](https://recovery.slingshot.filecoin.io/)
+- https://evergreen.slingshot.filecoin.io/ ⚠️
+- https://dashboard.slingshot.filecoin.io/ ⚠️
+
+## Filecoin+
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+To enhance the Filecoin's utility, [Filecoin Plus](https://filplus.storage/) introduced a social mechanism that incentivizes participants to maximize useful storage. Data owners used DataCap to make deals with storage providers, who benefited from increased block rewards over time. The Filecoin Plus program involved various stakeholders, including root key holders, notaries, data owners, and storage providers, who collaborated to form a governed community.
+
+## Open Panda
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+[Open Panda](https://openpanda.io/) is a platform for data researchers, analysts, students and enthusiasts to interact with the largest open datasets in the world. Data available through our platform is stored on Filecoin, a decentralized storage network comprised of thousands of independent Storage Providers around the world.
+
+## Data Program
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+[Data Program](https://dataprograms.org) listed tools, products, and incentive programs designed to drive growth and make data storage on Filecoin more accessible. This list is currently more up-to-date.
+
+## Moonlanding
+
+Moon Landing was designed to ramp up storage providers in the Filecoin network by enabling them to serve verified deals at scale.
+
+See also: https://moon-landing.io/ ⚠️
+
+## Filecoin Dataset Explorer
+
+![maintenance-status](https://img.shields.io/badge/maintenance-deprecated-red.svg)
+
+[Filecoin Dataset Explorer](https://datasets.filecoin.io/) showcased data stored on the Filecoin network between 2020 and 2022, including telemetry, historical archives, Creative Commons media, entertainment archives, scientific research, and machine learning datasets. It highlighted Filecoin's capability to store large datasets redundantly, ensuring availability from multiple Storage Providers worldwide. Each dataset is identified by a unique content identifier (CID). The platform aimed to make diverse datasets accessible to users globally.
+
+See also: [Legacy Explorer](https://legacy.datasets.filecoin.io/)
+
+## Estuary
```

Comment:
## Estuary (Estuary.tech)

*fix the tense. 
* Estuary was discontinued in July 2023, and the website shut down in April 2024. 
remove the note about documentation, we'll redirect that here as well.